integration/app/test-demo/src/components/MeetingEndControl.tsx

```diff
@@ -0,0 +1,24 @@
+import { ControlBarButton, Remove, useMeetingManager } from 'amazon-chime-sdk-component-library-react';
+import React from 'react';
+import { endMeeting } from '../utils/api';
+
+const MeetingEndControl = () => {
+  const meetingManager = useMeetingManager();
```

Comment:
No reason to test the meeting end workflow. Already covered in JS SDK

Reply:
Deleted all meeting end related code.